test/test_scanner.py

```diff
@@ -109,13 +118,22 @@ def _product_version_detection_test(self, url, filename, package, version):
         self.scanner = Scanner(cvedb=self.cvedb, should_extract=True)
         # run the tests
         self.scanner.recursive_scan(tempfile)
+        cves = self.scanner.all_cves
+        # make sure we found the expected product/version
+        # list of all the products
+        products = [cve_data["product"] for cve_data in cves]
+        # Make sure the product is in the results
+        assert product in products
+
+        product_versions = []
+        for cve_data in cves:
+            if cve_data["product"] == product:
+                product_versions.append(cve_data["version"])
 
-        # make sure we found the expected package/version
-        assert package in self.scanner.all_cves
-        assert version in self.scanner.all_cves[package]
+        assert version in product_versions
 
     @pytest.mark.parametrize(
-        "binary, package, version, are_in, not_in",
+        "binary, product, version, are_in, not_in",
```

Comment:
We aren't returning product from checker.  We are returning module name. You should revert this changes and just rename package to module in this case.